purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallColor.kt

```diff
@@ -24,6 +25,12 @@ data class PaywallColor(
     @RequiresApi(Build.VERSION_CODES.O)
     val underlyingColor: Color?,
 ) {
+    /**
+     * The color converted to a ColorInt
```

Comment:
Maybe
```suggestion
     * The color converted to a @ColorInt to be used by Compose
```

or something like that?

Reply:
I believe the part of `to be used by Compose`, is not necessary since this is basically an int that can be used in different places, not only for Compose. We can add the `@` though, to make it clearer that this `int` represents a `@ColorInt`.